### DISCUSSION TITLE
Swag providers + shippers

### DISCUSSION THREAD

**Discussion by alexcg1** [todogroup/ospology#6] [OSPO Discussions]
### Our problem

We've recently started [our swag program](https://jina.ai/2021/07/12/swag.html) and have had 90 swag signups in the first month. That's great! Except now we're faced with actually packing, labelling, and shipping all the stuff from our two global offices. And that's really leeching my team's productivity.

### My question

What swag providers/shippers/fulfillers do other companies use? I think that's the only solution that will scale.

### Context

- Many of our users are in India, so having a local provider there to reduce shipping cost and import duties is a high priority
- We have two tiers of swag at present:
  - Stickers + fridge magnet: Can fit into an envelope
  - Notepad, tote bag, keyring, plus above: Needs an actual package
- We've got a stockpile of swag at our global offices now. We need to work out what to do with it if we go for external provider :facepalm: 

Thanks for any help anyone can give!

**Comment by caniszczyk** [comment]
From my experience, I've used services like printfection.com and stickermule.com in the past and it serves the needs for the basics, even though it's albeit expensive.

In CNCF, we have a large scale store that is run through https://www.pinnaclebranding.com which handles all the inventory, distributions and running of the store, on top of helping find custom swag. It has scaled well for our 100K+ developer community but the downsides are that everything is in the east coast US so shipping costs can be expensive for folks internationally https://store.cncf.io

**Comment by jdorfman** [comment]
Hey @alexcg1, for Curiefense we use [Real Thread Fullfilment](https://www.realthread.com/fulfillment).

They make our t-shirts (high quality, screen printed) and we order our stickers from Sticker Mule and ship it to them.

We then set up a Google Form ([example](https://forms.gle/kafJXjk9Ayc3qwBR9)), add them as an editor, and then they update the spreadsheet with the status and tracking number (just in case you need it).

![Image 2021-08-04 at 10 01 19 AM](https://user-images.githubusercontent.com/398230/128223349-12cf81a1-29bf-40ab-a5d3-e5e9b5e2bd15.jpg)

So anyone who fills out that form, gets a t-shirt and two stickers. You can add other things besides stickers if you want, just ship it to them.

They have shipped to India, Israel,  Bangladesh, Portugal, and other countries for us.

Can't recommend them enough. If you want an intro to our account manager please LMK and I'll gladly facilitate that for you.
